### ISSUE TITLE
PCM 4.0 stuck at logo screen after MIB install and patch

### ISSUE THREAD

**Issue by VTEKKEN** [Mr-MIBonk/M.I.B._More-Incredible-Bash#392]
**Describe the bug**
HELP PLEASE!
I downloaded MIB and had it auto install (Swdlautorun.txt) into my MHI2, after that I was able to get into GEM and patched aa/cp. I then plugged my android device in and it was recognized as Carplay, however, no AA content appeared, but the AA logo was initially on. Then the AA logo also went away.  I then tried re patching AA/CP for POG24, however after doing that it rebooted and stayed in the Porsche logo screen, it is not taking any inputs at all.  I pulled the fuse, and the battery, but this still did not fix the issue. 

**Add M.I.B LOG file to your report**
[MU0984-JTB-07429.03.171192078A.log](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/files/13998819/MU0984-JTB-07429.03.171192078A.log)

**Screenshots**
![20240118_222959](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/157153507/4b57656f-d8bd-4c99-b01b-752f59388fa4)

**Additional context**
1) Unit is stock POG24, working well before, no know updates made to it prior to me owning it. 
2) Thing I did wrong that could have caused it after relooking at the instruction: SD card is a small one with adapter. 
3) I tried a full size SD with new image and was able to see that MIB did get reinstalled based on seeing the log and backup files filled. 
the Porsche logo stays on now even with the car off and locked, I'm keeping the fuse pulled to keep the battery from draining.

**Comment by VTEKKEN** [comment]
I also used the backup function prior to patching AA/CP, so I do have the original backup files, but can not get to GEM where I can reload the stock images.

**Comment by Cloudmania4** [comment]
Issue fixed?you have ax88772 based lan usb?

**Comment by Cloudmania4** [comment]
Fixed?

**Comment by cascoarc** [comment]
I haven't solved the problem...
I have the ax88772 board and Putty
What command should I use to solve the problem?

Thanks

**Comment by Cloudmania4** [comment]
Can you enter the telegram group ?

**Comment by cascoarc** [comment]
yes

**Comment by cascoarc** [comment]
I'm already

**Comment by cascoarc** [comment]
VAG-MIB2

**Comment by cascoarc** [comment]
your nick on telegram?

**Comment by cascoarc** [comment]
Yes

Cloudmania4 ***@***.***> escreveu em seg., 7/10/2024 às 11:30
:

**Comment by cascoarc** [comment]
Your nick on Telegram?

Cloudmania4 ***@***.***> escreveu em seg., 7/10/2024 às 11:30
:

**Comment by Cloudmania4** [comment]
Write in group hello

**Comment by HansiNEA** [comment]
Could you add me too?
My nickname is HansiNEA
THANK YOU

**Comment by yratush** [comment]
have you fixed it?

**Comment by cascoarc** [comment]
Yes 👍

yratush ***@***.***> escreveu em sex., 22/08/2025 às 19:36 :

**Comment by Eugene53705** [comment]
> Yes 👍
> 
> yratush ***@***.***> escreveu em sex., 22/08/2025 às 19:36 :
> […](#)

I have the same issue. However, the crash happened while uploading "full screen patch" while using "wide screen patch".

Could you maybe help me on this issue?

**Comment by Cloudmania4** [comment]
Login over dlink, and remove thejust the jar which is not working on that old fw!

**Comment by mv-go** [comment]
Hey! Same issue here. Freezes on the loading "Porsche" screen. It's a panamera. Tried accessing the file system via telnet and dlink dongle - no luck. Can't even ping the system. Tried rebooting and accessing via telnet immediately after the start - no luck. In adapter stats in windows it looks like there's "nothing" on the car end. The adapter and laptop are good - tried on Cayenne with mh2p - works fine. 5F looks operational via piwis, I can read the errors, do coding, etc. Any help?

**Comment by Cloudmania4** [comment]
Login and remove that fucking jar file for widescreen.no one will help you here find right person)

**Comment by mv-go** [comment]
> Login and remove that fucking jar file for widescreen.no one will help you here find right person)

Yeah, I know I gotta remove it. Apparently the dlink dongle just died on me. Will get another one and try telnet again :)
